tutorials/llm/llama-3/llama3-lora-nemofw.ipynb

```diff
@@ -0,0 +1,586 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d3323204-1463-4df3-8c75-5e95b6d66ba1",
+   "metadata": {},
+   "source": [
+    "# Creating a Llama-3 LoRA adapter with NeMo Framework"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29f3d632-44a0-4e6c-9229-b70bbcff1e99",
+   "metadata": {},
+   "source": [
+    "This notebook showcases performing LoRA PEFT **Llama-3 8B** on [PubMedQA](https://pubmedqa.github.io/) using NeMo Framework. PubMedQA is a Question-Answering dataset for biomedical texts."
```

Comment:
It might be worthwhile to call attention to the fact that you should be running this notebook inside a NeMo Framework container to make sure people have success in this notebook!

Reply:
fixed